_ecosystem/pytorchvideo

```diff
@@ -0,0 +1,10 @@
+---
+layout: ecosystem_detail
+title: PyTorchVideo
+summary: A deep learning library for video understanding research.
```

Comment:
Could we elaborate it more i.e, "A deep learning library for video understanding research. Hosts various video-focused models, datasets, training pipelines and more."

Reply:
Updated..